etl/extractData.py

```diff
@@ -1,68 +1,138 @@
-import json
-import pandas as pd
+import argparse
 import csv
-import yaml
 import os
-import logging
+
+import archive
+import pandas as pd
 from gen3.auth import Gen3Auth
-from gen3.query import Gen3Query
-from jsonobject import *
+from gen3.submission import Gen3Submission
 
-logging.basicConfig(
-    level=logging.DEBUG,
-    format="%(asctime)s %(levelname)s %(name)s %(message)s",
+parser = argparse.ArgumentParser(description="N3C Crosswalk Packaging")
+parser.add_argument(
+    "-cf",
+    "--crosswalk-file",
+    action="store",
+    type=str,
+    required=True,
+    help="N3C crosswalk file with Datavant tokens",
+)
+parser.add_argument(
```

Comment:
I see there are lot of args can you please add a comment to run this and what are mandatory args